include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -1282,6 +1288,20 @@ class concatenated_sequences
         return data() >= rhs.data();
     }
     //!\}
+
+    /*!\cond DEV
+     * \brief Serialisation support function.
+     * \tparam archive_t Type of `archive`; must satisfy seqan3::cereal_archive_concept.
+     * \param archive The archive being serialised from/to.
+     *
+     * \attention These functions are never called directly, see \ref serialisation for more details.
+     */
+    template <cereal_archive_concept archive_t>
+    void CEREAL_SERIALIZE_FUNCTION_NAME(archive_t & archive)
```

Comment:
No, every header file is required to include platform so that setups without cereal get the fallback definitions. Setups with cereal get `<cereal/cereal.hpp> ` by including `<seqan3/core/concept/cereal.hpp>` – which they need to include anyway for `cereal_archive_concept`.